src/accelerate/accelerator.py

```diff
@@ -2834,6 +2833,8 @@ def get_state_dict(self, model, unwrap=True):
                         "To save the full checkpoint, run `model.save_checkpoint(save_dir)` and use `zero_to_fp32.py` to recover weights."
                     )
             else:
+                from deepspeed.checkpoint.utils import clone_tensors_for_torch_save
```

Comment:
I am wondering what is ther relation between `self.deepspeed_config["zero_optimization"]["stage"] == 3` and `deepspeed == 0.9.3`.

The PR description says:

> This PR changes the place of such import so that the version error is raised appropriately.

But isn't the goal to avoid such `import` and `usage` of `clone_tensors_for_torch_save` when `deepspeed < 0.9.3`?